samples/oauth/index.html

```diff
@@ -64,9 +64,8 @@
         scope: '<Your OAuth Scopes>', // string; e.g. 'openid profile me.read'
         serverConfig: {
           baseUrl: '<Your AM URL>', // string; e.g. 'https://openam.example.com:9443/openam/'
-          timeout: '<Your Timeout (milliseconds)>' // number; 90000 or less
+          timeout: '<Your Timeout (milliseconds)>' // number; 3000 to 5000 is good, this impacts the redirect time to login
```

Comment:
I'd remove "this impacts the redirect time to login" as that is only relevant for central login. This sample app represents embedded login with OAuth, so there's technically no redirect to login.